app/components/ItuneCard/index.tsx

```diff
@@ -33,15 +36,49 @@ const StyledImg = styled.img`
   }
 `;
 
-const ItuneCard = ({ trackId, artistName, artworkUrl100, collectionName }: ItuneCardProps) => {
+const StyledAudio = styled.audio`
+  && {
+    width: 100%;
+    padding: 5px;
+  }
+`;
+
+const StyledMeta = styled(Meta)`
+  && {
+    padding: 5px;
+  }
+`;
+
+const ItuneCard = ({
+  trackId,
+  artistName,
+  artworkUrl100,
+  collectionName,
+  previewUrl,
+  currentTrackId,
+  setCurrentTrackId
+}: ItuneCardProps) => {
+  const audioRef = useRef(new Audio(previewUrl));
```

Comment:
Do we have to create a Audio element here ? Isn't passing audioRef to ref enough ?